fast64_internal/oot/actor/properties.py

```diff
@@ -145,15 +151,19 @@ def draw_props(self, layout: UILayout, altRoomProp: OOTAlternateRoomHeaderProper
 
 
 class OOTTransitionActorProperty(PropertyGroup):
-    roomIndex: IntProperty(min=0)
+    fromRoom: PointerProperty(type=Object, poll=lambda self, object: self.isRoomEmptyObject(object))
+    toRoom: PointerProperty(type=Object, poll=lambda self, object: self.isRoomEmptyObject(object))
     cameraTransitionFront: EnumProperty(items=ootEnumCamTransition, default="0x00")
     cameraTransitionFrontCustom: StringProperty(default="0x00")
     cameraTransitionBack: EnumProperty(items=ootEnumCamTransition, default="0x00")
     cameraTransitionBackCustom: StringProperty(default="0x00")
-    dontTransition: BoolProperty(default=False)
+    isRoomTransition: BoolProperty(name="Is Room Transition", default=False)
```

Comment:
I think this default needs to flip; the old version was "doors are room transitions unless dontTransition is selected".